### DISCUSSION TITLE
Welcome to Pokemon-GO-Friend-Codes Discussions!

### DISCUSSION THREAD

**Discussion by czantoine** [czantoine/Pokemon-GO-Friend-Codes#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KamiSamaKun** [comment]
Bonjour est il possible de mettre à jour le fichier api Google Sheets v3 API de votre projet svp il sont en v4 actuellement.
Bien à vous.

**Reply by czantoine** [reply]
Bonjour, 
Effectivement, il faudrait debug ça. Je te propose d'ouvrir une issue ou de faire une pull-request s'il te plait :)
